charts/README.md

```diff
@@ -282,15 +282,15 @@ schedulerExtender:
       monitor:
         enabled: true
 EOF
-helm install azuredisk-csi-driver azuredisk-csi-driver/azuredisk-csi-driver --namespace kube-system --version v2.0.0-alpha.1 --values /tmp/azure-isk-csi-driver-overrides.yaml
+helm install azuredisk-csi-driver azuredisk-csi-driver/azuredisk-csi-driver --namespace kube-system --version v2.0.0-beta.4 --values /tmp/azure-isk-csi-driver-overrides.yaml
```

Comment:
Should be `azuredisk-csi-driver-overrides.yaml` to match the name of the file we create above.